OracleWebLogic/dockerfiles/14.1.1.0/Dockerfile.slim-8

```diff
@@ -0,0 +1,110 @@
+ENV ORACLE_HOME=/u01/oracle \
+    USER_MEM_ARGS="-Djava.security.egd=file:/dev/./urandom" \
+    PATH=$PATH:${JAVA_HOME}/bin:/u01/oracle/oracle_common/common/bin:/u01/oracle/wlserver/common/bin
+
+# Setup filesystem and oracle user
+# Adjust file permissions, go to /u01 as user 'oracle' to proceed with WLS installation
+# ------------------------------------------------------------
+RUN mkdir -p /u01 && \
+    chmod a+xr /u01 && \
+    useradd -b /u01 -d /u01/oracle -m -s /bin/bash oracle
+
+# Environment variables required for this build (do NOT change)
+# -------------------------------------------------------------
+ENV FMW_PKG=fmw_14.1.1.0.0_wls_quick_slim_Disk1_1of1.zip \
+    FMW_JAR=fmw_14.1.1.0.0_wls_quick_slim.jar
+
+# Copy packages
+# -------------
+COPY $FMW_PKG install.file oraInst.loc /u01/
+RUN chown oracle:oracle -R /u01
+
+# Install
+# ------------------------------------------------------------
+USER oracle
+
+RUN cd /u01 && ${JAVA_HOME}/bin/jar xf /u01/$FMW_PKG && cd - && \
+    ${JAVA_HOME}/bin/java -jar /u01/$FMW_JAR -silent -responseFile /u01/install.file -invPtrLoc /u01/oraInst.loc -jreLoc $JAVA_HOME -ignoreSysPrereqs -force -novalidation ORACLE_HOME=$ORACLE_HOME && \
+    rm /u01/$FMW_JAR /u01/$FMW_PKG /u01/install.file && \
+    rm -rf /u01/oracle/cfgtoollogs
+
+# Final image stage
+FROM oracle/serverjre:8
+
+ENV ORACLE_HOME=/u01/oracle \
+    USER_MEM_ARGS="-Djava.security.egd=file:/dev/./urandom" \
+    SCRIPT_FILE=/u01/oracle/createAndStartEmptyDomain.sh \
+    HEALTH_SCRIPT_FILE=/u01/oracle/get_healthcheck_url.sh \
+    PATH=$PATH:${JAVA_HOME}/bin:/u01/oracle/oracle_common/common/bin:/u01/oracle/wlserver/common/bin
+
+# Domain and Server environment variables
+# ------------------------------------------------------------
+ENV DOMAIN_NAME="${DOMAIN_NAME:-base_domain}" \
+    ADMIN_LISTEN_PORT="${ADMIN_LISTEN_PORT:-7001}" \
+    ADMIN_NAME="${ADMIN_NAME:-AdminServer}" \
+    ADMINISTRATION_PORT_ENABLED="${ADMINISTRATION_PORT_ENABLED:-true}" \
+    ADMINISTRATION_PORT="${ADMINISTRATION_PORT:-9002}" \
+    DEBUG_flag=false \
+    PRODUCTION_MODE=prod
+
+# Setup filesystem and oracle user
+# Adjust file permissions, go to /u01 as user 'oracle' to proceed with WLS installation
+# ------------------------------------------------------------
+RUN mkdir -p /u01 && \
+    chmod a+xr /u01 && \
+    useradd -b /u01 -d /u01/oracle -m -s /bin/bash oracle && \
+    chown oracle:oracle /u01
+
+COPY --from=builder --chown=oracle:oracle /u01 /u01
+
+# Copy scripts
+#-------------
+COPY container-scripts/createAndStartEmptyDomain.sh container-scripts/create-wls-domain.py container-scripts/get_healthcheck_url.sh /u01/oracle/
+
+RUN chmod +xr $SCRIPT_FILE $HEALTH_SCRIPT_FILE && \
+    chown oracle:oracle $SCRIPT_FILE /u01/oracle/create-wls-domain.py $HEALTH_SCRIPT_FILE
+
+USER oracle
+
+HEALTHCHECK --start-period=10s --interval=10s --timeout=120s  --retries=12 CMD curl -k -s --fail `$HEALTH_SCRIPT_FILE` || exit 1
```

Comment:
These `HEALTHCHECK` values seem too large. It would take 12x120s (24m) for a service to fail. I would recommend reducing tthe `timeout` to ~5-10s and the `retries` to 3-5.